docs/advanced/gpu.md

```diff
@@ -0,0 +1,296 @@
+:::
+
+3. Install Nvidia-Container-Toolkit
+
+- If the edge node can directly access the external network, it can be installed directly according to the [official documentation](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html).
+- If the edge node cannot directly access the external network, you need to download the official [offline installation package](https://github.com/NVIDIA/nvidia-container-toolkit/releases) on a machine with network connectivity, 
+  and transfer the installation package to the edge node to complete decompression. After decompression, the following files should appear in the directory:
+
+```shell
+   root@edgenode:~/release-v1.16.0-rc.1-experimental/packages/ubuntu18.04/amd64# ls
+   libnvidia-container1_1.16.0~rc.1-1_amd64.deb      libnvidia-container-tools_1.16.0~rc.1-1_amd64.deb      nvidia-container-toolkit-operator-extensions_1.16.0~rc.1-1_amd64.deb
+   libnvidia-container1-dbg_1.16.0~rc.1-1_amd64.deb  nvidia-container-toolkit_1.16.0~rc.1-1_amd64.deb
+   libnvidia-container-dev_1.16.0~rc.1-1_amd64.deb   nvidia-container-toolkit-base_1.16.0~rc.1-1_amd64.deb
+```
+
+Execute the following command in this directory to complete the installation:
+
+```shell
+   sudo apt install ./*
+```
+
+4. Configure container runtime to support GPU
+
+After successfully installing Nvidia-Container-Toolkit, you can use `nvidia-ctk` to configure each container runtime to support GPU.
+
+```shell
+# docker
+sudo nvidia-ctk runtime configure --runtime=docker --set-as-default
+# containerd
+sudo nvidia-ctk runtime configure --runtime=containerd --set-as-default
+``` 
+
+5. Restart container runtime
+
+Restart the container runtime and confirm whether GPU is supported.
+
+```shell
+# docker:
+systemctl daemon-reload && systemctl restart docker
+# Check whether the runtime is modified successfully.
+root@nano-desktop:~# docker info |grep Runtime
+ Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux nvidia runc
+ Default Runtime: nvidia
+
+# containerd:
+systemctl daemon-reload && systemctl restart containerd
+# Check whether the runtime is modified successfully.
+root@edgenode:~# cat /etc/containerd/config.toml |grep nvidia
+      default_runtime_name = "nvidia"
+        [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.nvidia]
+          [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.nvidia.options]
+            BinaryName = "/usr/bin/nvidia-container-runtime"
+```  
+
+
+Through the above steps, the edge node already has a GPU driver, and the container runtime also has the ability to allocate GPU devices. Next, the edge node can be managed into the KubeEdge cluster.
+
+### Edge GPU node management
+
+Hosting edge GPU nodes mainly includes the following steps:
+
+1. Manage the node to the cluster
```

Comment:
same here can we just point the reference to https://kubeedge.io/docs/setup/install-with-keadm/ to avoid the redundancy and possible maintenance problem?